Cue/app/tabs/search/SearchHome.js

```diff
@@ -1,64 +1,113 @@
 // @flow
 
 import React from 'react'
-import { View, Text, Navigator, Platform } from 'react-native'
+
+import { searchDecks } from '../../actions'
+import type { State } from '../../reducers/user'
+import type { DeckMetadata } from '../../api/types'
+
+import { View, Text, Navigator, Platform, TextInput, Image  } from 'react-native'
 
 import { connect } from 'react-redux'
 
 import CueHeader from '../../common/CueHeader'
 import CueIcons from '../../common/CueIcons'
+import CueColors from '../../common/CueColors'
 
+import SearchListView from './SearchListView'
+
 const styles = {
   container: {
+    flex: 1,
+    backgroundColor: 'white'
+  },
+  bodyContainer: {
+    flex: 1,
+
+  },
+  searchBox: {
+    color: Platform.OS === 'android' ? 'white' : CueColors.primaryText,
+    backgroundColor: Platform.OS === 'android' ? 'transparent' : 'white',
+    borderRadius: 5,
     flex: 1,
-    backgroundColor: 'white'
+    fontSize: 16,
+    paddingLeft: 10,
+    paddingBottom: 6,
+    marginRight: Platform.OS === 'android' ? 90 : 16,
+    marginLeft: Platform.OS === 'android' ? 0 : 16,
+    height: Platform.OS === 'android' ? 50 : 35,
   },
-  bodyContainer: {
+  searchBoxContainer: {
     flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center'
-  }
+    flexDirection: 'row',
+    backgroundColor: 'transparent'
+  },
 }
 
+const placeHolderTextColor = Platform.OS === 'ios' ? 'white' : CueColors.lightGrey
```

Comment:
Oh woops, I mixed those two up. It should been white on android, light grey on ios.

I thought that light grey looked a bit weird on our blue header background.

What do you think?

White:
![white placeholder](https://cloud.githubusercontent.com/assets/7501968/24025325/10cee9c2-0a90-11e7-8703-4e8b63311c40.PNG)

Grey:
![grey placeholder](https://cloud.githubusercontent.com/assets/7501968/24025329/13e493aa-0a90-11e7-9b76-35ab26016afb.PNG)